src/core/RadioButton/base.ts

```diff
@@ -0,0 +1,5 @@
+import { RadioProps } from '@material-ui/core/Radio';
+export type RadioBaseProps = Omit<
+  RadioProps,
+  'children' | 'checked' | 'disabled'
+>;
```

Comment:
We are omitting these properties here and again using this in Radio.tsx. We won't need to do that since these are all required by default.

Reply:
@S-ayanide That is, we don't need Base file in that case?

Reply:
Yes, we don't